### PR TITLE
html-links | Keivan Mojmali

### PR DESCRIPTION
**Pull request by keivanmojmali**
image

### CODE REVIEW

File: html-links/index.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>HTML Links</title>
+    </head>
+    <body>
+
+      <h1>Popular Search Engines</h1>
+
+      <ul>
+        <a href="https://www.google.com"><li>Google</li></a>
```
Comment (Zhiwutian) [inline html-links/index.html:13]: very close! but you want to make sure that the `<a>` tags are nested within the `<li>` tags!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update needed

**Review by Zhiwutian — APPROVED** [review]
👍

File: html-links/index.html (hunk 1 above)
Comment (keivanmojmali) [reply html-links/index.html]: ahh I see.  that makes more sense now that I think about it. 
updated!

**Pull request merged**